packages/theme-base/src/themes/sap_fiori_3_dark/base-parameters.less

```diff
@@ -254,6 +254,10 @@
 @sapTile_IconColor: contrast(@sapTile_Background, lighten(@sapShellColor, 35), @sapContent_ContrastIconColor, @sapContent_ContrastTextThreshold);
 @sapObjectHeader_BorderColor: lighten(@sapObjectHeader_Background, 8);
 
-// custom added params as no base ("sap" prefix) params equivalent exist
 @sapToggleButton_Pressed_HoverBackground: darken(@sapSelectedColor, 3);
-@sapToggleButton_Pressed_HoverBorderColor: @sapToggleButton_Pressed_HoverBackground;
\ No newline at end of file
+@sapToggleButton_Pressed_HoverBorderColor: @sapToggleButton_Pressed_HoverBackground;
+
+@sapTimePicker_ItemBorderColor: @sapList_Background;
+@sapTimePicker_ItemColor: lighten(@sapList_Background, 3);
```

Comment:
We already have --sapList_Background_Darken3 ( darken(@sapList_Background, 3).
theme-base/src/themes/base/component-derived-colors.less
theme-base/src/themes/base/component-derived-colors-vars.less

So you can use it
@sapTimePicker_ItemColor: @sapList_Background_Darken3;

Reply:
Done